packages/block-editor/src/components/list-view/block-contents.js

```diff
@@ -56,7 +60,7 @@ const ListViewBlockContents = forwardRef(
 		} );
 
 		return (
-			<BlockDraggable clientIds={ [ block.clientId ] }>
+			<BlockDraggable clientIds={ selectedBlocks }>
```

Comment:
I don't think we can make this assumption in all cases. It's possible to start dragging a target in the list view sidebar/modal without selecting the block. (Block selection is keydown+keyup, while a drag start is just keydown).

https://user-images.githubusercontent.com/1270189/152205347-f7b20507-9e99-43bf-bf08-94bdfdb7e38f.mp4

Reply:
Great catch @gwwar, and thanks for all your detailed testing here. If I don't get to addressing all these issues today, I'll continue on working through them next week! 🙇

Reply:
I think I've come up with a fix for this. In https://github.com/WordPress/gutenberg/pull/38314/commits/91997f461e4b5492466108cc953f603c42eb1071 I've updated it so that the list of selectedBlocks is only passed in if the clicked on block is part of that selection, otherwise it uses the client id of the clicked on block. This appears to restore the behaviour where if no block is selected, dragging still works. It also appears to work correctly if you have a few blocks selected and then go to drag a block outside of that selection, but it probably needs a bit more manual testing to confirm 🙂